### PR TITLE
More robust looping of timestamp servers

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

**Review by gdams — APPROVED** [review]

File: serverTimestamp.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+comodaca=http://timestamp.comodoca.com/authenticode
+isectigo=http://timestamp.sectigo.com
+comodocarfc31611=http://timestamp.comodoca.com/rfc3161
```
Comment (adamfarley) [inline serverTimestamp.properties:3]: Is the final "1" a typo? Should it be "comodocarfc3161"?

Comment (karianna) [reply serverTimestamp.properties:3]: Good find! Fixed :-)

File: sign.sh [hunk 2]
```diff
@@ -62,23 +92,27 @@ signRelease()
...
-        if ! "$signToolPath" sign /f "${SIGNING_CERTIFICATE}" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.globalsign.com/scripts/timestamp.dll "$f"; then
-          echo "RETRYWARNING: Failed to sign ${f} at $(date +%T): Possible timestamp server error - RC $? ... Retrying in 10 seconds"
-          sleep 10s
-          "$signToolPath" sign /f "${SIGNING_CERTIFICATE}" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.globalsign.com/scripts/timestamp.dll "$f"
-        fi        
+        for ((i = 0; i < ${#TIMESTAMPKEYS[@]}; i++))
+        do
+          if ! "$signToolPath" sign /f "${SIGNING_CERTIFICATE}" /p "$SIGN_PASSWORD" /fd SHA256 /t "${TIMESTAMPVALUES[i]}" "$f"; then
+            echo "RETRYWARNING: Failed to sign ${f} at $(date +%T): Possible timestamp server error at \"${TIMESTAMPVALUES[i]}\" - RC $? ... Trying new server in 5 seconds"
+            sleep 5s
```
Comment (sxa) [inline sign.sh:99]: If we're just looping through different ones I'd argue this retry delay is probably unneccesary

Comment (karianna) [reply sign.sh:99]: I kept that in there in case of temporary localhost or local network issue - if that makes sense I can add a comment to that effect?

Comment (sxa) [reply sign.sh:99]: Yep happy with that, although at only seconds it would have to be very temporary :-)

**Comment by karianna** [comment]
rebase sucks

**Pull request closed without merging**